tests/__fixtures__/tracking-plan-fixture.json

```diff
@@ -207,6 +207,34 @@
         "required": ["properties"],
         "type": "object"
       }
+    },
+    {
+      "description": "This is JSON Schema draft-04 event.",
+      "name": "Draft-04 Event",
+      "rules": {
+        "$schema": "http://json-schema.org/draft-04/schema#",
+        "properties": {
+          "context": {},
+          "properties": { "type": "object" },
+          "traits": {}
+        },
+        "required": ["properties"],
+        "type": "object"
+      }
+    },
+    {
+      "description": "This is JSON Schema draft-06 event.",
+      "name": "Draft-06 Event",
+      "rules": {
+        "$schema": "http://json-schema.org/draft-06/schema#",
+        "properties": {
+          "context": {},
+          "properties": { "type": "object" },
+          "traits": {}
+        },
+        "required": ["properties"],
+        "type": "object"
+      }
```

Comment:
Should we include fixtures for draft 7, and also verify that it handles const and if/then/else?

Reply:
I'm planning on adding in more fixtures for "advanced" JSON Schema (especially around anyOf/etc.) in a later pass. Will add const/if/then/else ones too :)